cypress/e2e/UITests/pageObjects/MainChartPage.js

```diff
@@ -0,0 +1,29 @@
+class MainChartPage {
+  // Define elements and actions within the page
+  getSearchField() {
+    return cy.get('input[aria-invalid="false"]');
+  }
+
+  getChartNameFromResultsList() {
+    return cy.get('[class="MuiGrid-root MuiGrid-container MuiGrid-align-items-xs-center MuiGrid-justify-content-xs-space-between"]').children('[class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6"]');
```

Comment:
@georgepavlakos7 is this a stable selector? Is there another way to get the the elements from list?

Reply:
I cannot build the application at the moment to check, but there are a number a ways to make it more dynamic. Use indexer (0, 1, 2, etc) with partial element. Or check by text all the cells